scripts/github-release.ts

```diff
@@ -13,6 +13,7 @@ const publishRelease = async (version: string, commit: string) => {
     method: 'PATCH',
     body: JSON.stringify({
       prerelease: false,
+      make_latest: 'true',
```

Comment:
The type is `string` see https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#update-a-release

![Uploading image.png…]()